viewer/src/main/java/nl/b3p/viewer/stripes/UserLayerActionBean.java

```diff
@@ -134,6 +136,34 @@ public void stream(HttpServletResponse response) throws Exception {
         };
     }
 
+    public Resolution validate() throws FilterToSQLException, CQLException {
+
+        final UserLayerHandler ulh = new UserLayerHandler(auditMessageObject, Stripersist.getEntityManager(),
+                application, appLayer, query, title, wellKnownUserLayerWorkspaceName,
+                wellKnownUserLayerStoreName);
+
+        boolean success = true;
+        String isInvalidMsg = ulh.validate();
+        final JSONObject jsonObject = (new JSONObject()).put("success", Boolean.TRUE);
+        jsonObject.put("valid", isInvalidMsg);
+        jsonObject.put("sql", ulh.getSQLQuery());
```

Comment:
Wat moet de client met de SQL? die SQL mag iig niet meer terugkomen naar de server als input

Reply:
De client was ik die aan het debuggen was: ik wilde weten of de sql goed werd opgebouwd. Zal inderdaad weg kunnen